### DISCUSSION TITLE
How to use netcdf2grid programs properly?

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#219] [Q&A]
Hello developers,

When I used program "Netcdf2GriddData" I found little demonstration about this program, I don't konw how to convert ERA5 with 37 levels data to gridded data(output: lon lat time level pressure temperature ...). Could you please give me some advices about how to used this program? Thanks! 

regards,
Yongzhao Fan

**Comment by tmayerguerr** [comment]
Hi Yongzhao Fan,

GriddedData is a 2 dimensional data format (with multiple variables). The GriddedDataTimeSeries format has an additional time axis. But it is not possible to handle 3D data. NETCDF is more flexible than GROOPS. Therefore, not all possible NETCDF files can be converted. Therefore, at the moment I do not see how to read in ERA5 data.

regards
Torsten
